primitives/transaction/src/historic_transaction.rs

```diff
@@ -13,6 +18,84 @@ use crate::{
     Transaction as BlockchainTransaction,
 };
 
+/// The raw transaction hash is a type wrapper.
+/// This corresponds to the hash of the transaction without the execution result.
+/// This hash is the external facing hash.
+#[derive(Clone, Debug, Eq, PartialEq, Hash)]
+pub struct RawTransactionHash {
+    hash: Blake2bHash,
+}
```

Comment:
Why not using a `tuple struct`? 
```suggestion
pub struct RawTransactionHash(Blake2bHash)
```